ci-operator/jobs/openshift/release/openshift-release-release-4.2-periodics.yaml

```diff
@@ -1785,3 +1785,65 @@ periodics:
     - name: pull-secret
       secret:
         secretName: ci-pull-credentials
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  labels:
+    job-type: launch
+    job-env: openstack
+    job-release: '4.2'
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
```

Comment:
I added the pj-rehearse label here. This does make it the only job in this file with that label however. If it works maybe we drop it in a subsequent push.